Categories/categories.controller.js

```diff
@@ -0,0 +1,31 @@
+const { Category } = require("./categories.model.js")
+
+const getAllCategories = async (req, res) => {
+  try {
+    const categories = await Category.find({})
+    if (!categories) {
+      res.status(400).json({ success: false, message: "No categories found. Sorry!" })
+    }
+    res.status(200).json({ success: true, categories: categories })
```

Comment:
I think you missed a `return` statement here....   `return res.status(400).json({ success: false, message: "No categories found. Sorry!" })`